pennylane/pauli/pauli_arithmetic.py

```
@@ -249,19 +294,29 @@ def to_mat(self, wire_order, format="dense"):
         Rasies:
             ValueError: Can't get the matrix of an empty PauliSentence.
         """
+
+        def _pw_wires(w):
+            """To account for empty pauli_words which represent identity operations."""
+            if w:
+                return wires.Wires(w)
+
+            ps_wires = self.wires
+            if len(ps_wires) > 0:
+                return wires.Wires(
+                    list(ps_wires)[0]
+                )  # return any wire from the Pauli sentence's wires
+
```

Should we raise an error here?